008_team_6_agent_design/008_team_6_agent_design.tex

```diff
@@ -1 +1,1021 @@
+
+Despite the presence of selfish (and even narcissistic) agents in the tower, they all follow the rule dictated by the treaties they signed whilst being collectivist. \footnote{Thus simulation enforces \textit{honest} agents that do not have the ability to break a treaty before the expiration date. We leave the concept of deception for future work.} An issue with this configuration is that the exact day when the agents are eating is not synchronized, leading to the selfish and narcissistic agents eating 2 food as soon as their health falls below the \textit{weakLevel}. We hence conclude that not waiting in the critical zone as collectivists do leads to a non-zero number of deaths.
+
+In addition, we can also see the effect of the reshuffle period on the social motives distribution in \Cref{fig:res_D} (d). The reshuffle period is 30 days in this case and we see that the reorganization of the tower leads to some of the narcissistic agents updating their social motive to revert to selfish.
+
+The main observation to draw from these results is in the terminal behaviour and the capability to reach a stable steady state, with the distribution of all agent types tending to a continuous value (approximately 40\% collectivist, 40\% selfish and 20\% narcissist). We propose that it is this enforcement of treaties that allows for otherwise self-interested agents to act selflessly, instead acting towards a common goal of collectivism. This re-stabilisation of the initially homogeneous collectivist society through treaties is so powerful that it can support a substantial proportion of narcissistic agents, who have in previous experiments (A4, C4) been sufficient for disrupting the stability of a system even in low concentrations. We thus conclude that the inclusion of localised governance allows agents to transcend the limitations of impulse due to pre-defined social behaviours and self organise to maintain a stable system that supports all behavioural types.
+
+These results support the emergence of long-term stability, as stated in hypothesis 3, when using treaties. D1 shows a regular oscillatory behaviour whereas D2 shows a very constant distribution of the social motives.
+
+Finally, these simulations also clearly show that it is not explicitly necessary to inject external non-collectivist agents in the system to destabilize an otherwise purely collectivist system. Collectivists themselves, if they have the ability to change their social motives, will destroy an originally pure collectivist society. This result is even more drastic than hypothesis 4.
```

Comment:
```suggestion
Finally, these simulations also clearly show that it is not explicitly necessary to inject external non-collectivist agents in the system to destabilize an otherwise purely collectivist system. Collectivists themselves, if they have the ability to change their social motives, will destroy an originally pure collectivist society. This result is even more drastic than Hypothesis 4.
```

The last sentence is a bit weird to me